controller/k8s/api.go

```diff
@@ -787,6 +787,7 @@ func (api *API) GetServiceProfileFor(svc *corev1.Service, clientNs string) *spv1
 		}
 	}
 	// Not found; return default.
+	log.Infof("no Service Profile found for '%s' -- using default", dst)
```

Comment:
a quick check of `linkerd logs` shows this contributing to the majority of log lines. consider `log.Debugf`